pytorch_toolkit/formula_recognition/README.md

```diff
@@ -81,6 +81,36 @@ In addition to common parameters you can specify the following arguments:
 - `train_transforms_list` - similiar to `val_transforms_list`
 - `epochs` - number of epochs to train
 
+One can use some pretrained models. Right now two models are available:
+* medium model:
+    * [checkpoint link](https://download.01.org/opencv/openvino_training_extensions/models/formula_recognition/medium_photograped_0185.pth)
+    * digits, letters, some greek letters, fractions, trigonometric operations are supported; for more details, please, look at corresponding vocab file
+    * to use this model, please, change these fields in the config file:
+    ```
+    backbone_config:
+        arch: resnext50_32x4d
+        disable_layer_3: true
+        disable_layer_4: true
+    model_path: <path to the model>
+    vocab_path: vocabs/vocab_medium.json
+    ```
+The model can be used either for recognizing rendered formulas and for recognizing scanned formulas (e.g. from a scanner or from a phone camera)
+
+* handwritten polynomials model:
+    * [checkpoint](https://download.01.org/opencv/openvino_training_extensions/models/formula_recognition/polynomials_handwritten_0166.pth)
+    * digits, letters, upper indices are supported
+    * to use this model, please, change these field in the config file:
+    ```
+    backbone_config:
+        arch: resnext50_32x4d
+        disable_layer_3: false
+        disable_layer_4: true
+    model_path: <path to the model>
+    vocab_path: vocabs/vocab_handwritten_polynomials.json
+    ```
+The model can be used for recognizing handwritten polynomial equations.
+All the above models can be used for aftertuning or as ready for inference models. To provide maximum quality at recognizing formulas, it is highly recommended to preprocess image - simply binarize it, you can find corresponding prepocessing at [this file](im2latex/data/utils.py). Just state the desired preprocessing at the corresponding section of the config file (train, eval, etc).
```

Comment:
Once again, my understanding is that config file + checkpoint should be enough. No manual tweaking.

Reply:
For medium model it is true, but polynomials model differs from medium one not only in weights, but in architecture too. So, we have to change the configuration.

Reply:
Ok, take your time and think about it.

Reply:
What is your proposal? Create an additional config file for handwritten polynomials model?

Reply:
Added config for handwritten polynomials